src/client/dfs/dfs.c

```diff
@@ -355,7 +355,7 @@ fetch_entry(daos_handle_t oh, daos_handle_t th, const char *name,
 		*exists = true;
 
 out:
-	if (fetch_sym)
+	if (value)
```

Comment:
do you need the check?   D_FREE handles NULL

Reply:
This is one location that's reporting a lot, in fact there's something odd in the handling of value in this function which I'm going to fix-up elsewhere.  This line is about 20% of the DFS/dfuse logging on metadata lookups.